Tests/StencilTests/EnvironmentSpec.swift

```diff
@@ -32,9 +39,304 @@ func testEnvironment() {
 
       try expect(result) == "here"
     }
+
+    func expectedSyntaxError(token: String, template: Template, description: String) -> TemplateSyntaxError {
+      let range = template.templateString.range(of: token)!
```

Comment:
Always avoid implicit unwraps. You can replace this with a guard that falls back to an error with less information if the token isn't found. Or a `fatalError` (as this is a test, might as well break early).

Reply:
I would prefer failing here as it is crystal clear what is the reason than pollute the code with boilerplate guards especially considering that this is a test and we don't need graceful error handling or whatever.

Reply:
A `guard` with a `fatalError` and a message then. It's not because it's a test that it shouldn't be good code.

Reply:
I think this code is good enough, but I will do that if that's what it takes to merge this.

Reply:
@djbe done in 96a004e